configurator/src/cv.rs

```diff
@@ -146,6 +145,8 @@ impl Display {
 impl Drop for Display {
     fn drop(&mut self) {
         highgui::destroy_window(&self.window_name).unwrap();
+        // hack to get the window to close, see https://stackoverflow.com/a/50710994
+        highgui::wait_key(1).unwrap();
```

Comment:
I have so many questions, yet want to ask none of them. Just OpenCV things I guess...